examples/with-firebase/fetchData/getProfileData.js

```diff
@@ -0,0 +1,15 @@
+import admin from '../firebase/nodeApp'
+
+export const getProfileData = async (username) => {
+  const db = admin.firestore()
+  const profileCollection = db.collection('profile')
+  const profileDoc = await profileCollection.doc(username).get()
+
+  if (!profileDoc.exists) {
+    return null
+  }
+
+  const profileData = profileDoc.data()
+  const profileDataJson = JSON.stringify(profileData)
+  return profileDataJson
```

Comment:
This step is not required. Next.js will stringify the data if you return it as props 👍

Reply:
Thank you for your comment. Fixed!